### PR TITLE
resolve naming conflict cause by 'type' and 'titleColor'

### PR DESCRIPTION
**Pull request by iaulakh**
### Platforms Impacted
- [x] Android

### Description of changes
resolve naming conflict cause by 'type' and 'titleColor'

### Verification
tested demo app

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [x] VoiceOver and Keyboard Accessibility
- [x] Internationalization and Right to Left layouts
- [x] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

**Comment by aishwarya56** [comment]
Please raise it against develop

**Comment by iaulakh** [comment]
> Please raise it against develop

done, will resolve conflicts soon.

File: fluentui_others/src/main/res/values/attrs_action_bar_layout.xml [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-    <declare-styleable name="ActionBarLayout">
-        <attr name="type" format="enum">
+    <declare-styleable name="FluentUiActionBarLayout">
+        <attr name="fluentUiType" format="enum">
             <enum name="basic" value="0"/>
             <enum name="icon" value="1"/>
             <enum name="carousel" value="2"/>
         </attr>
-        <attr name="viewPager" format="reference"/>
+        <attr name="fluentUiViewPager" format="reference"/>
```
[the remark this answers is not in the document]
Comment (iaulakh) [reply fluentui_others/src/main/res/values/attrs_action_bar_layout.xml]: done, thanks

**Comment by michaelxiao16** [comment]
This issue has been resolved with Mayank's PR here https://github.com/microsoft/fluentui-android/pull/145

I will be closing this request without merging.

**Pull request closed without merging**